articles/users/search/v3/index.md

```diff
@@ -289,5 +176,3 @@ Search by date | `updated_at:<2018-01-15` | `updated_at:[* TO 2018-01-15}`
 Search by date | `last_login:<=2017-12` | `last_login:[* TO 2017-12]`
 String exact match | `name.raw:"john richard doe"` | `name:"john richard doe"`
 Phrase contains a word | `name:"richard"`, `name:richard` | `name:*richard*`
-
```

Comment:
Can you add a next-steps element at the end of this doc, listing these two?
articles/users/search/v3/query-syntax.md
articles/users/search/v3/best-practices.md

Add it on its own section, like `Keep reading`